mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1,3 +1,4 @@
+// ON CHANGING THESE, ALSO CHANGE Jenkinsfile.downstream
```

Comment:
Irrelevant to the code, are you planning on to remove the Jenkinsfile.downstream-xdlops in another PR?

Reply:
Yeah, I'll detele `Jenkinsfile.dowstream-xdlops` once we've validated that this works.